src/metabase/pulse/render/table.clj

```diff
@@ -22,6 +22,8 @@
     :padding-top     :20px
     :padding-bottom  :5px}))
 
+(def ^:private max-bar-width 106)
```

Comment:
The old way was to have the TDs fixed at `width: 106px` and the divs (each of which is a bar of the chart) as `width: x%`, where `x` is the normalized score.

The new way is to shrink the TDs to the smallest reasonable size (there's a `width: 1%` somewhere) and set the width of each div in pixels. A width of `x/106 px` is the same as a width of `x%` of a 106px container.

There's nothing particularly magic about 106, but that's the value that was used before and it seems like a fine size.